src/sql/workbench/services/notebook/browser/models/notebookModel.ts

```diff
@@ -1569,6 +1569,9 @@ export class NotebookModel extends Disposable implements INotebookModel {
 			changeType: change,
 			cells: [cell]
 		};
+		if (!cell.active) {
```

Comment:
I don't think we should we changing the active cell on every change - especially with the kernel support @corivera is adding (since that stuff can be long running and be changing output and whatnot)

And does this handle tabbing moving onto a different cell (but not editing it, just focusing on an element in a different cell)? For this stuff I think you should instead be looking into hooking into the tab events and updating the active cell as needed there instead.

Reply:
well I only update it when it's not already active - which should be done in any case. 

I didn't think of focusing the cell when one of its elements are tabbed into - for that I agree I need to hook into the tab event. I was only addressing the editing cell not being set as active.

Reply:
Yes, but just because the output (or even worse metadata) is changing doesn't mean that the cell is "active". And setting it to active can interrupt the user if they're currently doing something in another cell.

The issue that I filed is very specific to tabbing through different elements so I think we should be focused on fixing the issue for that alone and not trying to expand this beyond that scope unless there's specific reasons we think we need to address other situations.